zanata-war/src/test/resources/arquillian.xml

```diff
@@ -18,10 +18,6 @@
       <!--
                   <property name="javaVmArguments">-Xms1536m -Xmx1536m -XX:MaxPermSize=256m -Dorg.jboss.as.logging.per-deployment=false -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=8787</property>
       -->
-      <property
-        name="javaVmArguments">-Xms1536m -Xmx1536m -XX:MaxPermSize=256m -Dorg.jboss.as.logging.per-deployment=false
```

Comment:
I think it would be best to keep the `-Dorg.jboss.as.logging.per-deployment=false` (here and in the comment below) to make sure logging doesn't get switched off: http://stackoverflow.com/a/14575830/14379